src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -204,8 +205,61 @@ def child_sort_key(item: TraceEvent) -> List[int]:
 
 
 def query_trace_data(
-    trace_id: str, params: Mapping[str, str]
+    trace_id: str, params: Mapping[str, str], use_snql: bool = False
 ) -> Tuple[Sequence[SnubaTransaction], Sequence[SnubaError]]:
+    if use_snql:
+        transaction_query = QueryBuilder(
+            Dataset.Discover,
+            params,
+            query=f"trace:{trace_id}",
```

Comment:
Does this need `event.type:transaction`? Or is that implied by all these transaction-specific columns?

Reply:
Good catch, updated to `Dataset.Transactions`